lib/runPrettier.js

```diff
@@ -0,0 +1,61 @@
+const { spawn } = require('child_process');
+const path = require('path');
+
+const cwd = process.cwd();
+
+const filePattern = 'src/**/*.js';
+
+function runPrettier({ write, listDifferent, config }) {
+  const prettierConfig = config || {};
+
+  const prettierPath = path.join(
+    require.resolve('prettier'),
```

Comment:
This is now relative to require.resolve.
Still not ideal as any change to the depth of the `main` attribute in Prettier will break this implementation.